### DISCUSSION TITLE
macOS: LLVM ERROR: Failed to infer result type(s)

### DISCUSSION THREAD

**Discussion by connect-m** [vladmandic/sdnext#353] [Q&A]
Hi,

i just went throught the installation on my Mac Studio and after submitting my prompt I get this back on the console:

_Initializing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:-- 0:00:00loc("mps_add"("(mpsFileLoc): /AppleInternal/Library/BuildRoots/97f6331a-ba75-11ed-a4bc-863efbbaf80d/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShadersGraph/mpsgraph/MetalPerformanceShadersGraph/Core/Files/MPSGraphUtilities.mm":228:0)): error: input types 'tensor<2x1280xf32>' and 'tensor<1280xf16>' are not broadcast compatible
LLVM ERROR: Failed to infer result type(s).
Abort trap: 6_

Any ideas how to get it fixed?

Thanks!

Ronald

**Comment by ezam0000** [comment]
same here

**Comment by vladmandic** [comment]
anything M1/M2, lets continue at #347
